regression/verilog/part-select/indexed-part-select5.desc

```diff
@@ -0,0 +1,9 @@
+KNOWNBUG
+indexed-part-select5.sv
+--bound 0
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
+--
+This yields invalid SMT-LIB ("invalid extract application").
```

Comment:
For that to materialise, wouldn't also some `--smt` command line be required?

Reply:
We do run all tests both with the flattening backend and with Z3.